src/pages/new-todo/NewTodo.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const NewTodo = () => {
+  return (
+    <div>NewTodo</div>
```

Comment:
What's the difference between NewTodo and TodoCreate which was deleted?

Reply:
No diff, just wrote what came to mind and didn't copy:)